app/theme/client/main.css

```diff
@@ -58,3 +58,429 @@
 
 /* RTL */
 @import 'imports/general/rtl.css';
+
+/* customerChat History CSS */
+.inner-addon {
+	position: relative;
+            left: 24px;
+
+                    width: 345px;
+                        height: 40px;
+
+                        box-sizing: border-box;
+                
+                    border: 2px solid #cbced1;
+
+        background: #ffffff;
+
+	border-radius: 2px;
+}
+
+.inner-addon .glyphicon {
+	position: absolute;
+	            bottom: 0.5%;
+        
+        color: #2f343d;
+    
+	padding: 10px;
+    
+            pointer-events: none;
+	
+}
+
+.searchresults {
+            margin-top: 7%;
+}
+
+.search-li {
+    display: block;
+        
+            padding-bottom: 3%;
+        position: relative;
+
+                height: 65px;
+    
+    background-color: #ffffff;
+}
+
+.search-li :hover {
+
+            cursor: default;
+}
+
+.closed-div {
+    position: relative;
+                top: 20px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5378317)